src/store/rootReducer.js

```diff
@@ -2,8 +2,8 @@
 import { combineReducers } from 'redux';
 
 // modules
-import demo from 'modules/demo';
+import articleVote from 'modules/vote';
```

Comment:
Maybe the module should also be called `articleVote` since there'll be `commentVote`